docker-compose.yml

```diff
@@ -236,7 +366,10 @@ services:
     ports:
       - "80:80"
       - "8090:8080"
-      - "9200:443"
+      - "9200:9200"
+      - "9980:9980"
+      - "8880:8880"
+      - "9981:443"
```

Comment:
@SagarGi if we provide a comment about which service is bound to which port will it be too much?